sw/device/lib/dif/dif_entropy.c

```diff
@@ -17,10 +17,8 @@
  */
 static void set_config_register(const dif_entropy_t *entropy,
                                 const dif_entropy_config_t *config) {
-  // TODO: Make this configurable at the API level.
-  // TODO: Currently bypass disable cannot be set, as it causes the hw fsm to
-  // get stuck
-  // uint32_t reg = 0;
+  // TODO: Currently bypass disable cannot be cleared, as it causes the hw fsm
+  // to get stuck.
   uint32_t reg =
```

Comment:
Is this idea here that since the chip has already booted and the boot phase has already occurred, that this test intentionally sets the disable bypass bit to just test the "startup" and "continuous" phases of ES?

Reply:
Also, if there is a functional issue here, an issue should be written up.